### PR TITLE
Create an endpoint to keep track of the number of clicks

### PR DESCRIPTION
**Pull request by fegzycole**
This PR creates an endpoint that redirects a user when he pastes the shortened link in the browser and sends the request.

It also updates the count of a particular link after each visit

### CODE REVIEW

File: src/controllers/getURL.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Url from '../models/UrlShorten';
+
+const getUrlAndEditCount = async (req, res) => {
```
Comment (shonubijerry) [inline src/controllers/getURL.js:3]: Can we rename this function to `updateUrlClickCount`

File: src/controllers/getURL.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  try {
+    const url = await Url.findOne({ urlCode: req.params.urlCode });
+    if (url) {
+      url.click_count += 1;
+      await url.save();
+      return res.redirect(url.long_url);
+    } else {
```
Comment (shonubijerry) [inline src/controllers/getURL.js:10]: There is no need wrapping this statement in an else block since we already returned a response inside the if block.

**Review by shonubijerry — CHANGES_REQUESTED** [review]
You have done a good job, kindly attend to the comments so this PR can be approved for merging.

File: src/controllers/getURL.js (hunk 1 above)
Comment (fegzycole) [reply src/controllers/getURL.js:3]: The thing is that two things are happening in the route. We are updating the count and then we are redirecting the trimmed url to the original url, that was the reason for the naming.

Comment (shonubijerry) [reply src/controllers/getURL.js:3]: Okay

File: src/controllers/getURL.js (hunk 2 above)
Comment (shonubijerry) [reply src/controllers/getURL.js:10]: Attend to this then so we can merge

Comment (fegoworks) [reply src/controllers/getURL.js:10]: I will be attending to this shortly

**Review by shonubijerry — APPROVED** [review]

**Pull request merged**